src/servers/Cargo.toml

```diff
@@ -34,7 +35,7 @@ num_cpus = "1.13"
 once_cell = "1.16"
 openmetrics-parser = "0.4"
 opensrv-mysql = "0.3"
-pgwire = "0.6.1"
+pgwire = "0.6.2"
```

Comment:
```suggestion
pgwire = "0.6"
```

Reply:
here we require a minimum version 0.6.2 to trigger an update on Cargo.lock because previous versions have known bug.